src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+}
+
+.goods {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0;
+  width: $conteiner-width;
+}
```

Comment:
if its a separate block it should have a separate file with corresponding name, or just make it as element